cli/src/cmd/forge/script.rs

```diff
@@ -0,0 +1,932 @@
+use crate::{
```

Comment:
everything looks good - need to spend some more time testing it / reviewing this file, but overall very exciting. might be time we got a "nice" abstraction for linking, because it seems to complicate things a lot in this case?